superset/assets/javascripts/explore/components/ExploreViewContainer.jsx

```diff
@@ -32,18 +33,33 @@ const propTypes = {
 class ExploreViewContainer extends React.Component {
   constructor(props) {
     super(props);
+    this.firstLoad = true;
+    this.actionLog = null;
     this.state = {
       height: this.getHeight(),
       width: this.getWidth(),
       showModal: false,
     };
   }
 
+  componentWillMount() {
+    this.actionLog = new ActionLog({
+      action: LOG_ACTIONS_LOAD_EXPLORE,
+      containerId: this.props.slice.slice_id,
```

Comment:
do you think this should just be `id`? it's not exactly a container. cc @john-bodley 

also what if an explore view doesn't have a slice id/isn't saved?

Reply:
in existed logging logic, for explore view without existed slice_id, the log entry will have slice_id=0